EAD/Session9/Assignment/MetaParkingSystemV2/src/main/java/com/metacube/metaparkingsystemv2/utils/ConnectionUtility.java

```diff
@@ -0,0 +1,48 @@
+package com.metacube.metaparkingsystemv2.utils;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+import org.springframework.context.annotation.PropertySource;
+
+@PropertySource("classpath:application.yml")
+public class ConnectionUtility 
+{
+	private String driverPath = "com.mysql.cj.jdbc.Driver";

```

Comment:
We can database.properties to take driverPath,Username and Password for database